src/Console/GeneratePaletteCommand.php

```
@@ -22,115 +21,121 @@ class GeneratePaletteCommand extends Command
-		return implode( ' ', $parts );
-	}
+    protected $description = 'Generate an accessible color palette from seed colors.';
+
+    public function handle(): int
+    {
+        $primary = $this->option('primary');
+        $secondary = $this->option('secondary');
+        $size = (int) ($this->option('size') ?: 9);
+        $format = $this->option('json') ? 'json' : strtolower((string) ($this->option('format') ?: 'md'));
+        $output = $this->option('output');
+        $strictness = strtoupper($this->option('strictness') ?: 'AA');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$strictness'. (undefined)

(UnusedLocalVariable)